provider/k8s/app.go

```
@@ -52,6 +52,19 @@ func (p *Provider) AppCreate(name string, opts structs.AppCreateOptions) (*struc
 		return nil, errors.WithStack(err)
 	}
 
+	sourceSecret, _ := p.Cluster.CoreV1().Secrets(p.RackName+"-system").Get(context.TODO(), "registries", am.GetOptions{})
+	if sourceSecret != nil {
+		newSecret := &ac.Secret{
+			ObjectMeta: am.ObjectMeta{
+				Name:      sourceSecret.Name,
+				Namespace: p.AppNamespace(name),
+			},
+			Type: sourceSecret.Type,
+			Data: sourceSecret.Data,
+		}
+		p.Cluster.CoreV1().Secrets(p.AppNamespace(name)).Create(context.TODO(), newSecret, am.CreateOptions{})
```

why are you ignoring error?